docs/content/en/migration.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: 'Migration guides'
-description: ''
+description: 'Migrate with confidence from one version to another with our migrations guides.'
```

Comment:
Migration guides and notes are mentioned in changelog/releases. This section is only for breaking changes (and kinda legacy now)

Reply:
Added the `major`  prefix